libs/utils/utils.js

```diff
@@ -152,11 +152,19 @@ const TARGET_TIMEOUT_MS = 4000;
 const LANGSTORE = 'langstore';
 const PREVIEW = 'target-preview';
 const PAGE_URL = new URL(window.location.href);
+const LANGUAGE_BASED_DOMAINS = [
+  // don't add milo too. It's a special case because of tools, merch, etc.
+  'news.adobe.com',
+  '--news--adobecom.',
+];
```

Comment:
Can we rely on a flag set by consumers? Bleeding consumer implementation details back into milo is an anti-pattern. (Which I'm guilty of myself)

If consumers set this in their scripts, they can onboard to language based by adding a feature-config in their project without any PR to milo.

Reply:
I really wish we could. 
There are two use cases to consider in my opinion. News.adobe.com is about to be a language based site, and so when you're on news.adobe.com, we need to know to convert the links to language based links. 
When you're on business.adobe.com though and you have links pointing to news.adobe.com, when they get localized they need to be localized directly. That means if I'm on business.adobe.com/ch_de/, my links pointing to news.adobe.com, need to point to news.adobe.com/de/ch/ or at the very least news.adobe.com/de/

Reply:
I see, unfortunate. 🫤

Reply:
What about languages that news.adobe.com does not support, e.g. business.adobe.com/it -> news.adobe.com/it ?

Reply:
>When you're on business.adobe.com though and you have links pointing to news.adobe.com, when they get localized they need to be localized directly.

Can't that be solved through the mapping used in `loadLanguageConfig` ? https://main--federal--adobecom.aem.page/federal/assets/data/languages-config.json